internal/pwru/types.go

```diff
@@ -110,6 +112,7 @@ func (f *Flags) SetFlags() {
 	flag.StringVar(&f.Backend, "backend", "",
 		fmt.Sprintf("Tracing backend('%s', '%s'). Will auto-detect if not specified.", BackendKprobe, BackendKprobeMulti))
 
+	flag.Uint32Var(&f.SetPerCPUBuf, "set-percpu-buf", 4096, "set the size of the percpu buffer")
```

Comment:
Maybe you could be more specific with the flag description? I.e. instead of saying "percpu buffer", you could say "set the size of buffers to print skb data (used by --output-skb and --output-skb-shared-info)".